### DISCUSSION TITLE
why can't I enable visual python from terminal from my MacBook

### DISCUSSION THREAD

**Discussion by ghost** [visualpython/visualpython#190] [Q&A]
why can't I enable visual python from terminal from my MacBook by using visualpy install command, it shows that /Library/Frameworks/Python.framework/Versions/3.10/bin/visualpy: line 258: pip: command not found
and
cp: /visualpython: No such file or directory

**Comment by minjk-bl** [comment]
Hello @StarCraftFANS :)
Is there any anaconda/jupyter environment you have? Visual Python need Jupyter environment(with python 3).
